calico/felix/fetcd.py

```diff
@@ -290,10 +290,16 @@ def _load_config_dict(self):
         # profiles by id.  The response contains a generation ID allowing us
         # to then start polling for updates without missing any.
         global_cfg = self.client.read("/calico/config/")
-        host_cfg = self.client.read("/calico/host/%s/config/" %
-                                    self.config.HOSTNAME)
+        configs = [global_cfg]
+        try:
+            host_cfg = self.client.read("/calico/host/%s/config/" %
+                                        self.config.HOSTNAME)
+        except KeyError:
+            _log.info("No configuration overrides for this node")
+        else:
+            configs.append(host_cfg)
 
-        for child in itertools.chain(global_cfg.children, host_cfg.children):
+        for child in itertools.chain(*[cfg.children for cfg in configs]):
```

Comment:
This feels like overkill, given that we don't ever expect to have more configs. Why not just set `host_cfg` to the empty list before the try...except, and then keep the rest of the code the same?